design-documents/graph-ql/coverage/customer/gift-registry.graphqls

```diff
@@ -129,26 +134,55 @@ input GiftRegistryDynamicAttributeInput {
 }
 
 type CreateGiftRegistryOutput {
+    status: Boolean!
+    create_gift_registry_user_errors: [GiftRegistryItemsUserError]!
```

Comment:
What about having some general UserError type, and move the `status` and `user_errors` into a new OutputInterface, so each Output type is going to implement this interface?

```suggestion
    user_errors: [GiftRegistryItemsUserError]!
```

Reply:
> What about having some general UserError type, and move the `status` and `user_errors` into a new OutputInterface, so each Output type is going to implement this interface?
So I did, except there's a difference between gift_registry_user_errors and gift_registry_item_user_errors
One is an array and the the one is not. Check it out on the latest version